packages/ses/src/error/tame-error-constructor.js

```diff
@@ -74,6 +87,55 @@ export default function tameErrorConstructor(
     setPrototypeOf(NativeError, SharedError);
   }
 
+  // https://v8.dev/docs/stack-trace-api#compatibility advises that
+  // programmers can "always" set `Error.stackTraceLimit`
+  // even on non-v8 platforms. On non-v8
+  // it will have no effect, but this advice only makes sense
+  // if the assignment itself does not fail, which it would
+  // if `Error` were naively frozen. Hence, we add setters that
+  // accept but ignore the assignment on non-v8 platforms.
+  defineProperties(InitialError, {
+    stackTraceLimit: {
+      get() {
+        if (typeof OriginalError.stackTraceLimit === 'number') {
+          // OriginalError.stackTraceLimit is only on v8
+          return OriginalError.stackTraceLimit;
+        }
+        return undefined;
+      },
+      set(newLimit) {
+        if (typeof OriginalError.stackTraceLimit === 'number') {
+          // OriginalError.stackTraceLimit is only on v8
+          OriginalError.stackTraceLimit = newLimit;
```

Comment:
If someone does `Error.stackTraceLimit = undefined` or `= 'oops'` or something surprising, would this preclude them from ever changing it back again? Node.js doesn't seem to complain when you assign a non-Number, at least not right away.

Fixing that might require a separate flag (recorded early, before anything has a change to modify `Error.stackTraceLimit`), which remembers whether we're on a platform that has it or not, rather than checking each time.

Reply:
Changed so that if `newLimit` is not a number, it is silently ignored. Added a comment explaining why.